src/history/api.py

```diff
@@ -91,6 +95,23 @@ def _clean_code(code: str) -> str:
 CourseData = dict[str, Union[str, ClassDict]]
 
 
+def get_round_numbers(year: Union[str, int]) -> tuple[int]:
```

Comment:
Small nitpick - upon running `./scripts/checks.sh`, `mypy` complains about the following:
```
src/history/api.py:112: error: Incompatible return value type (got "tuple[int, ...]", expected "tuple[int]")  [return-value]
```
The correct type signature should return a `tuple[int, ...]` and everything else should be fine.

Reply:
Oops, my bad.